src/style.css

```diff
@@ -0,0 +1,18 @@
+@import "./styles/card.css";
+@import "./styles/stars.css";
+
+:root {
+  --main-color: #060b35;
+  --extra-color: #616070;
+  --button-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
don't set `margin` for all elements. please just use specific needed seletors
```suggestion
```

Reply:
You can reset margin only for those tags, you need
p, ul etc...

Reply:
fixed